cmd/esbuild/service.go

```diff
@@ -649,6 +658,13 @@ func (service *serviceType) convertPlugins(key int, jsPlugins interface{}) ([]ap
 					}
 				}
 
+				if args.IsSectionFocusActive && args.Kind == api.ResolveJSDynamicImport && !isFocused(args) {
+					resolvedPath := filepath.Join(args.ResolveDir, args.Path);
+					return api.OnResolveResult{
+						Path: resolvedPath + ".es-async",
```

Comment:
If the `onDynamicImport` PR is rejected, look into using a namespace to "tag" dynamic imports.